### PR TITLE
support DTS for Ceiling

### PR DESCRIPTION
**Pull request by andrejsokolov**
Ticket - #-42233

### CODE REVIEW

File: inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp [hunk 1]
```diff
@@ -108,6 +108,7 @@ const Transformations& getDefaultTransformations() {
         {ngraph::opset3::Convert::type_info,           dynamicToStaticUnaryElementwise},
         {ngraph::opset3::Clamp::type_info,             dynamicToStaticUnaryElementwise},
         {ngraph::opset3::Floor::type_info,             dynamicToStaticUnaryElementwise},
+        {ngraph::opset3::Ceiling::type_info,           dynamicToStaticUnaryElementwise},
```
Comment (AndrewBakalinIntel) [inline inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp:111]: It's better to use the latest opset `opset5`.

File: inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_ceiling.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+protected:
+    std::shared_ptr<ngraph::Node> createTestedOp() override {
+        const auto& parameters = GetParam();
+        const auto& inDataType = std::get<0>(parameters);
+        const auto& inDataShapes = std::get<1>(parameters);
+        targetDevice = std::get<2>(parameters);
+
+        const auto inputSubgraph = createInputSubgraphWithDSR(inDataType, inDataShapes);
+
+        const auto ceiling = std::make_shared<ngraph::opset3::Ceiling>(inputSubgraph);
```
Comment (AndrewBakalinIntel) [inline inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_ceiling.cpp:31]: Oh, it seems like I was wrong when I said it's similar to clamp in our offline discussion. They are almost the same, but the reason why `Clamp` is extracted into separate test class is that it has additional parameters in its constructors unlike other postops (better to compare `Ceiling` with `Floor`).
So just put `Ceiling` operation into test instantiations in `dsr_unary_elementwise.cpp` & `dynamic_to_static_shape_unary_elementwise.cpp` test files.

Comment (gladilov-gleb) [reply inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_ceiling.cpp:31]: BTW, I don't see a unit test (that compares function vs function)

File: inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp (hunk 1 above)
Comment (andrejsokolov) [reply inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape.cpp]: Fixed

File: inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_ceiling.cpp (hunk 2 above)
Comment (andrejsokolov) [reply inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_ceiling.cpp]: Fixed

**Review by AndrewBakalinIntel — APPROVED** [review]

**Review by gladilov-gleb — APPROVED** [review]

**Comment by gladilov-gleb** [comment]
@Maxim-Doronin could it be merged?

**Review by Maxim-Doronin — APPROVED** [review]

**Pull request merged**